tests/functional/test_install_check.py

```diff
@@ -2,9 +2,8 @@
 
 
 def assert_contains_expected_lines(string, expected_lines):
-    lines = string.splitlines()
     for expected_line in expected_lines:
-        assert any(line.endswith(expected_line) for line in lines)
+        assert expected_line in string
```

Comment:
```suggestion
        assert expected_line + "\n" in string
```

Reply:
Indeed, it's `assert_contains_expected_lines` after all.